src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,232 @@
-              <IconButton
-                aria-label="Areas"
-                className={classes.roundButton}
-                color={key.startsWith('/areas') ? 'primary' : 'secondary'}
-                data-test="area-button"
-                size="large"
-              >
-                <Map />
-              </IconButton>
+              <Avatar
+                alt="icon"
+                src={`/api/orgs/${orgId}/avatar`}
+                sx={{ alignSelf: 'center', marginLeft: 0.5 }}
+              />
             </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
-              >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
+          )}
+          {open && (
+            <ZUIFuture future={model.getOrganization(orgId)}>
+              {(data) => {
+                return (
+                  <Box alignSelf="center">
+                    <Box display="flex" justifyContent="start">
+                      {hover ? (
+                        <IconButton onClick={toggleDrawer}>
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
+                            sx={{ alignSelf: 'center', marginLeft: 2 }}
+                          />
+                        </IconButton>
+                      ) : (
+                        <Avatar
+                          alt="icon"
+                          src={`/api/orgs/${orgId}/avatar`}
+                          sx={{ alignSelf: 'center', marginLeft: 2 }}
+                        />
+                      )}
+                      <Typography gutterBottom m={1} variant="h6">
+                        {data.title}
+                      </Typography>
+                    </Box>
+                  </Box>
+                );
+              }}
+            </ZUIFuture>
+          )}
         </Box>
-      </List>
+        <Divider />
+        <List>
+          {menuItemsMap.map((item) => (
+            <ListItem
+              key={item.name}
+              disableGutters
+              disablePadding
+              sx={{
+                display: 'block',
+              }}
+            >
+              <NextLink href={`/organize/${orgId}/${item.name}`} passHref>
+                <IconButton
+                  className={classes.roundButton}
+                  size="large"
+                  sx={{
+                    justifyContent: open ? 'initial' : 'center',
+                    minHeight: 48,
+                    ml: open ? 1 : 2,
+                    mr: open ? 2 : 1,
+                    px: 2.5,
```

Comment:
To me this styling seems so complicated, and mixing `sx` and `className` makes it difficult to know where styling comes from. Lets try to refactor this together.